### DISCUSSION TITLE
github campus expert problems

### DISCUSSION THREAD

**Discussion by slancerk** [community/community#35094] [GitHub Education] [answered]
**Received my github campus expert mail today on 04/10/2022 but the deadline is showing 02/10/2022, what to do?**
![WhatsApp Image 2022-10-04 at 20 16 23](https://user-images.githubusercontent.com/88985517/193855452-88007ca9-0f08-480d-89e5-9e6f66c00a10.jpeg)

**Comment by samus-aran** [comment, marked as answer]
Hey @Slancerk :wave: , [this thread](https://github.com/community/community/discussions/34984) should provide more clarity on the incident. Sorry for any inconvenience caused.
